index.html

```diff
@@ -791,6 +791,36 @@ <h3>Devices</h3>
           </dd>
         </dl>
     </section>
+    <section>
+      <h3>Generic</h3>
+        <p>
+The generic domain contains use cases that can apply to any or all of the proceeding domains
+        </p>
+        <dl class="left dl-horizontal">
+          <dt>
+<udef>E.1 Non-transferability of credentials</udef>
+          </dt>
+          <dd>
+            <p>
+Many different types of plastic cards today have the words "Non-transferable" or similar printed on them. Examples include:
```

Comment:
I think it would be useful/helpful if it would be described more explicitly why in certain credential types non-transferability is valuable, in addition to what kind of credentials have that property.

Reply:
To give you a definitive answer, one would need to ask the existing issuers of plastic cards as to why they inserted this property onto their cards. The fact that they do is not in dispute. 
One can hypothesise as to the use cases by looking at the specific cards that have these properties. In the case of my historical cathedral card the intended use case is that only the card holder should be able to enter the cathedral free of charge and that the original card holder (i.e. the issuee) should not give the card to anyone else. In the case of credit cards the use cases are self explanatory: only the subject of the credit card is authorised to use it anywhere where the card can be used.
Does this help?